docs/installing-open-mpi/configure-cli-options/required-support-libraries.rst

```diff
@@ -1,21 +1,25 @@
-.. _label-building-ompi-cli-options-support-libraries:
+.. _label-building-ompi-cli-options-required-support-libraries:
 
-Support libraries
-^^^^^^^^^^^^^^^^^
+CLI Options for required support libraries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-:ref:`See this section <label-install-required-support-libraries>` for
-information about the CLI options for the support libraries described
-in this section.
+See the :ref:`Required support libraries
+<label-install-required-support-libraries>` section for information
+about the required support libraries described in this section.
 
-The following are command line options for the support libraries that
-are used by Open MPI that can be used with ``configure``:
+The following are command line options for the required support
+libraries that are used by Open MPI that can be used with
```

Comment:
This is a little wordy, maybe:

```
The following are command line options for the required support
 libraries that are used by Open MPI with `configure``:
 ```

Reply:
Good suggestion.  I ended up going even more concise.  😄